### PR TITLE
Misc100

### PR DESCRIPTION
**Pull request by morebrownies**
**Changes**
- Changed Mac tray icon
- Changed Windows tray icon
- Added tooltip to long listings in the discover section (more than 30 chars)
- Blur background when transaction modal is opened now (required me to move the modal element to index.html)
- Prevent body scroll when transaction modal is open
- Cleaned up the double cell border issue in the store listing view
- Prevent more than two decimal places in the fee input (not sure how great my JS is for this)
- Darkened tooltips to prevent collisions with body content

### CODE REVIEW

**Comment by morebrownies** [comment]
@jjeffryes ready for review

File: js/views/transactionsVw.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ module.exports = baseVw.extend({
       orderID: options.orderID,
       status: options.status,
       serverUrl: this.serverUrl,
-      parentEl: this.$el.find('.js-transactionModalHolder'),
+      parentEl: $('.js-transactionModalHolder'),
```
Comment (jjeffryes) [inline js/views/transactionsVw.js:258]: Why move this to index? It should stay in the transaction view, that way if you navigate away from that view it's automatically removed. 

Are there other modals or elements we want the transaction modal to be layered on top of?

File: js/views/pageNavVw.js [hunk 2]
```diff
@@ -350,18 +350,19 @@ module.exports = Backbone.View.extend({
     $('#obContainer').addClass('blur');
 
     // display the modal
-    $('.js-aboutModal').removeClass('hide');
+    $('.js-aboutModal').removeClass('fadeOut').addClass('fadeIn');
   },
```
Comment (jjeffryes) [inline js/views/pageNavVw.js:181]: We'll have to watch this, previously the fadeIn/fadeOut classes caused problems. 

It should also just need one class, fadeOut, removing it should do all you need.

Comment (morebrownies) [inline js/views/pageNavVw.js:181]: I was trying to get the modal to fadeIn like the transaction modal, but didn't have much luck.

Comment (jjeffryes) [inline js/views/pageNavVw.js:181]: I can change it to fade in like the other modal.

Comment (morebrownies) [inline js/views/pageNavVw.js:181]: It would be nice to get them all working the same way if it's relatively simple.

**Pull request merged**